PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,23 @@
+Thank you for contributing to JanusGraph.
```

Comment:
To avoid adding yet more files to the root of the repo, what about moving this file to a `.github` subdirectory, since this is a GitHub-specific workflow? We might also add an `ISSUE_TEMPLATE.md` in the future, so this will help keep the root of the repo leaner.

https://github.com/blog/2111-issue-and-pull-request-templates says both top-level files and files in `.github` are supported.

I would recommend keeping `CONTRIBUTING.md` at the root of the repo since it's independent of GitHub and applicable to everyone.

Thoughts?

Reply:
@mbrukman It was originally in .github but was moved back as part of review (see [this comment](https://github.com/JanusGraph/janusgraph/pull/146#discussion_r109833851) and [this comment](https://github.com/JanusGraph/janusgraph/pull/146#issuecomment-318944995)). I'm okay with changing it back if that makes more sense.

Reply:
Sorry, missed the original thread at the time it was pointed out, and looks like it wasn't quite resolved with a consensus. Added a comment on that thread as well with my proposal.

Reply:
Should I move the PULL_REQUEST_TEMPLATE.md back into the .github folder?

Reply:
@mbrukman Is it okay to just move it back to .github? I'm definitely okay with it (I just want this PR finally merged). @kottmann Sorry I moved it in the first place.

Reply:
@kottmann, @sjudeng — yes, please.